Services/ChatHub.cs

```diff
@@ -18,7 +28,10 @@ public Task SendMessageToCaller(string user, string message)
         }
 
         public Task SendGroupMessage(string groupId, string user, string message)
-        {
+        {   
+            _messageService.AddMessage(message, Context.UserIdentifier, groupId);
```

Comment:
@SimonasKalpokas1 IdBasedUserIdentifier is used right here by SignalR when I call for Context.UserIdentifier.

Reply:
Thanks, now I get it.